ocp_resources/deployment.py

```diff
@@ -21,7 +21,8 @@ def scale_replicas(self, replica_count=int):
         Returns:
             Deployment is updated successfully
         """
-        body = super().to_dict()
+        super().to_dict()
+        body = self.res
```

Comment:
No need to save `body = self.res`, update `self.res` directly.

Reply:
done